tests/models.py

```diff
@@ -668,6 +668,24 @@ def expressions(self, create: bool, extracted: list[Any], **kwargs: Any) -> None
         if create:
             db.session.commit()
 
+    @factory.post_generation  # type: ignore[misc]
+    def _references(self: "Group", create: bool, extracted: list[Any], **kwargs: Any) -> None:
+        if not create:
+            return
+
+        _validate_and_sync_component_references(
+            self,
+            ExpressionContext.build_expression_context(
+                collection=self.form.collection, fallback_question_names=True, mode="interpolation"
+            ),
+        )
+
+        # Wipe the cache of questions on a form - because we're likely to be creating more forms/questions
+        del self.form.cached_questions
+
+        if create:
+            db.session.commit()
```

Comment:
tiniest nit ever on a fixture but this should always be true as this looks like a no-op at the top when its just building